# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/base/crashtests/crashtests.list
# Commit: dd4da257c754
# Full Hash: dd4da257c754cb3b93295e1eb5f1c0421513c155
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2022-03-16 09:52:31
# Description:
#   Bug 1759631 - Back out the crashtest for bug 1749190. r=tnikkel
#   
#   This test runs a lot of JS and triggers a lot of unrelated assertions,
#   some of which are fatal. The test added in bug 1755790 is much simpler
#   and a better fit for providing test coverage for the patches in bug
# ==============================================================================

diff -r 2430eeb36caf -r dd4da257c754 layout/base/crashtests/crashtests.list
--- a/layout/base/crashtests/crashtests.list	Wed Mar 16 05:56:26 2022 +0000
+++ b/layout/base/crashtests/crashtests.list	Wed Mar 16 06:00:27 2022 +0000
@@ -563,7 +563,6 @@
 load 1729578.html
 load 1729581.html
 load 1734007.html
-asserts-if(Android,0-1000) load 1749190.html
 load 1745860.html
 pref(layout.accessiblecaret.enabled,true) load 1746989.html
 load 1752649.html
